index.html

```diff
@@ -9,16 +9,33 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
+  <div id="wide-menu" class="wide-menu hidden">
+    <div class="flex-right">
+      <button type="button" class="img-btn">
+        <img src="icons/Icon-Cancel.png" alt="Cancel">
+      </button>
+    </div>
+    <div class="container-2 column">
+      <a href="#headline"><h2>Portfolio</h2></a>
+      <div class="divider-1"></div>
+      <a href="#about-myself"><h2>About</h2></a>
+      <div class="divider-1"></div>
+      <a href="#contact"><h2>Contact</h2></a>
+      <div class="divider-1"></div>
```

Comment:
- [ ] Instead of creating a `div` in HTML just to make it serve as a line (divider), it's better if you add those lines using CSS by making the anchor (`a`) tags a certain width and then setting their `border-bottom` and adjusting the spaces using `padding-bottom` 💯